### DISCUSSION TITLE
HookWebpackError: Cannot read properties of undefined (reading 'e')

### DISCUSSION THREAD

**Discussion by tumenasarturas** [webpack/webpack#17242] [Q&A] [answered]
Solved

**Comment by alexander-akait** [comment]
Please run `npm ls webpack`

**Reply by alexander-akait** [reply]
As you can see you have `webpack@5.83.0`, you need to update all deps to `webpack@5.83.1`

**Reply by alexander-akait** [reply]
Because you don't have `webpack` directly in your `package.json`, so yarn install the new version and move old version inside packages, you need to update deps (or remove and install again deps to regenerate lockfile)

**Reply by tumenasarturas** [reply, marked as answer]
Thank you
